Lib/test/test_unittest/testmock/testmock.py

```diff
@@ -1547,6 +1547,8 @@ def f(x=None): pass
         mock = Mock(spec=f)
         mock(1)
 
+        uncalled_mock = Mock()
```

Comment:
Can you please move the object closer to the place where it is used?

Reply:
Done in https://github.com/python/cpython/pull/113573/commits/1e178389978b12aa85ff3cd1de16c6e17b45e89a